src/pages/PromptPage/PromptPage.scss

```diff
@@ -157,4 +158,9 @@
     margin-right: 10px;
   }
 
+  &__bottom-section {
```

Comment:
I think there's an extra border here
![Screen Shot 2024-05-02 at 8 09 45 PM](https://github.com/makeitMVPadmin/Content/assets/146603843/7c722710-f33f-43b0-a818-3468553eb510)

Reply:
Whoops, good catch. Sorry about that.